src/Compilers/Core/Portable/SourceGeneration/SyntaxStore.cs

```diff
@@ -69,13 +73,12 @@ public IStateTable GetSyntaxInputTable(SyntaxInputNode syntaxInputNode, NodeStat
                         else
                         {
                             syntaxInputBuilders.Add((node, node.GetBuilder(_previous._tables, _enableTracking)));
+                            _syntaxTimes[node] = TimeSpan.Zero;
```

Comment:
>_syntaxTimes[node] = TimeSpan.Zero; [](http://example.com/codeflow?start=28&length=35)

Should the values in `_syntaxTimes` be initialized in the constructor rather than here?